easybuild/easyconfigs/f/FoX/FoX-4.1.2-intel-2017b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'FoX'
+version = '4.1.2'
+
+homepage = 'http://homepages.see.leeds.ac.uk/~earawa/FoX/'
+description = """FoX is an XML library written in Fortran 95.
+It allows software developers to read, write and modify XML documents from Fortran applications without the
+complications of dealing with multi-language development."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'usempi': False, 'openmp': False}
+
+sources = [
+    SOURCE_TAR_GZ,
+]
+
+source_urls = [
+    'http://homepages.see.leeds.ac.uk/~earawa/FoX/source/'
+]
```

Comment:
@Micket Please include a SHA256 checksum, and collapse re-order `source_urls` and `sources`:

```
sources = ['...']
sources = [SOURCE_TAR_GZ]
checksums = ['...']
```

(`eb --inject-checksums FoX-4.1.2-intel-2017b.eb`) will do that for you.